ui/src/pages/QuickSetup/RequestServicesPage.tsx

```diff
@@ -46,23 +58,24 @@ const RequestServicesPage = (props: { adminCredentials: Credentials }) => {
   const [creatingRequest, setCreatingRequest] = useState(false);
   const [addedSuccessfully, setAddedSuccessfully] = useState(false);
 
-  const provider = requestInfo?.provider;
+  const customer = requestInfo?.customer;
 
   useEffect(() => {
-    if (provider) {
+    if (customer) {
       if (isHubDeployment) {
-        setToken(userParties.find(p => p.party === provider)?.token);
+        setToken(userParties.find(p => p.party === customer)?.token);
       } else {
-        setToken(computeToken(provider));
+        setToken(computeToken(customer));
       }
     }
-  }, [userParties, provider]);
+  }, [userParties, customer]);
 
   const onFinishCreatingRequest = (success: boolean) => {
     if (success) {
       setAddedSuccessfully(true);
       setTimeout(() => {
         setAddedSuccessfully(false);
+        setRequestInfo(undefined);
```

Comment:
resetting request info so that I can display messages like "this service has already been requested!" given newly-added operator view of all services